### ISSUE TITLE
Non convergence of Loss function

### ISSUE THREAD

**Issue by DuXikb** [Li-Chongyi/Zero-DCE#54]
Is this model a problem of non convergence of Loss function with training process? Do I need to change the hyperparameters when I use the training set that comes with the project and does not converge?
